### ISSUE TITLE
Node IDs exporting as taxonomy IDs causing errors

### ISSUE THREAD

**Issue by Elapworth** [mjordan/islandora_workbench#772]
I used the 'get data from view' task but some fields that appear as empty in the CSV file actually have data in them in the site. Files are attached. For example, for node 410168, field_archival_collection is empty in the CSV but on the site it has the value https://special.library.unlv.edu/node/1662. The warning I get in the log is: 

16-Apr-24 08:19:35 - INFO - Exporting data for node 410168 "Photograph of Valley View Golf Club members".
16-Apr-24 08:19:35 - WARNING - Term ping (GET) on https://special.library.unlv.edu/taxonomy/term/1662?_format=json returned a 404 status code.

Config file:
task: get_data_from_view
host: "https://special.library.unlv.edu"
view_path: '/admin/digital-collections/iwe/json'
view_parameters:
    - 'field_digital_id_value=ohr'
username: elapworth-workbench
password: 
content_type: dc_object
export_csv_file_path: /Users/elapworth/Downloads/ohrexport3.csv
export_csv_field_list: ['field_digital_id','field_archival_collection','title','field_physical_identifier','field_citation','field_archival_resource_key','field_edtf_date','body','field_source','field_member_of','field_staff_note','field_digital_provenance']
log_file_path: /Users/elapworth/Downloads/ohrexport3.log

[ohrexport3.csv](https://github.com/mjordan/islandora_workbench/files/15030372/ohrexport3.csv)
[ohrexport3.log](https://github.com/mjordan/islandora_workbench/files/15030374/ohrexport3.log)

**Comment by mjordan** [comment]
Something similar has come up in the past, where (IIRC) the ID of the term that caused the error belonged to a deleted term, but the ID was still registered with the node. Before I dig into this to confirm (and fix), I want to point out that the node ID and the term ID are the same (1622). Can you confirm this is the case? Seems like that would be unlikely but not impossible. But I want to make sure I understand what's going on in this situation before looking for a fix. Thanks.

**Comment by Elapworth** [comment]
Yes, the ID numbers are the same, but there is no 1662 taxonomy term. Workbench should be returning the node ID for that field (https://special.library.unlv.edu/node/1662) but instead it seems like it is looking for a taxonomy term for some reason, which I think is why I get the error in the log file. Although node IDs for other rows in the CSV file exported correctly for that field.

**Comment by mjordan** [comment]
Hunch: the following code in workbench_fields.py should return `None` if `ping_term()` is False: https://github.com/mjordan/islandora_workbench/blob/main/workbench_fields.py#L944-L946 . I'll test that.

**Comment by mjordan** [comment]
@Elapworth can you checkout the issue_772 branch and try your task again?

**Comment by mjordan** [comment]
OK, thanks, I'll need to investigate further. Similar report in #775.
